src/components/Modal/index.js

```diff
@@ -22,7 +22,7 @@ class Modal extends Component {
         overlayClassName={style.overlay}
         bodyClassName={style.modalBody}
         className={style.inner}
-        shouldCloseOnOverlayClick={true}
+        shouldCloseOnOverlayClick={shouldCloseOnOverlayClick !== false}
```

Comment:
I think this `shouldCloseOnOverlayClick` property would be better defined as a a default prop for the component class, like this
```
static defaultProps = {
    shouldCloseOnOverlayClick: true
}
```
Then you wouldn't need to do `shouldCloseOnOverlayClick !== false` to maintain the current modal behaviour as the default.

Reply:
applied